### ISSUE TITLE
IPTV

### ISSUE THREAD

**Issue by telase** [casperb123/VideoPlayer#1]
the program is nice but why doesn't it play iptv

**Comment by casperb123** [comment]
Yes i know that it can only play media files. I only made it because i wanted to try to make a video player. It wasn’t really intended to be released to everyone. I’m probably not going to update it anymore, only if there is some serious bugs.
